src/pycatcher/catch.py

```diff
@@ -231,7 +231,7 @@ def detect_outliers(df: pd.DataFrame) -> Union[pd.DataFrame, str]:
     if df_pandas.index.is_unique:
         # Find the time frequency (daily, weekly etc.) and length of the index column
         inferred_frequency = df_pandas.index.inferred_freq
-        logging.info("Time frequency: ", inferred_frequency)
+        logging.info("Time frequency: %s", inferred_frequency)
```

Comment:
Ah! I guess I added this bug 🙄

Reply:
No Jagadish, I had added that logging :)